output.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Can we find a better place for this to live? The project root is not the place for this file.

Reply:
@intl-man Agree, dispensation module would be a better place for it. Thanks!